docs/AboutUs.md

```diff
@@ -9,15 +9,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Chin Zhe Ning
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/chinzhening.png" width="200px">
```

Comment:
Your image name should be your github username in lower case.

Reply:
Thanks for pointing that out, it has been updated.